src/main/cliapp/src/utils/utils.js

```diff
@@ -226,3 +226,32 @@ export function generateInitialTableState(columns, defaultVisibleColumns){
 	}
 	return { defaultColumnNames, initialTableState };
 }
+
+export function validateBioEntityFields(updatedRow, setUiErrorMessages, event, setIsEnabled, closeRowRef, areUiErrors) {
+	const bioEntityFieldNames = ["subject", "diseaseGeneticModifier", "sgdStrainBackground" ];
```

Comment:
We should also add "assertedAllele" to this.  Entering text into the "with" and "assertedGenes" fields doesn't throw an error but just removes the text on saving - I don't know if we can deal with those fields in the same way due to the multi-select nature of those columns, but the current behaviour may be confusing (probably something for a separate ticket though)

Reply:
@markquintontulloch "assertedAllele" has been added to the validation list